icarus_backend/drone/testDroneViews.py

```diff
@@ -70,4 +70,18 @@ def test_get_drones_past_missions(self):
         gdpm_json = {'drone_id': '1', 'mission_id': '1'}
         response = self.client.post(reverse('get drones past missions'), json.dumps(gdpm_json),
                                     content_type='application/json')
-        self.assertEqual(response.status_code, 200)
\ No newline at end of file
+        self.assertEqual(response.status_code, 200)
+
+    def test_edit_drone_details(self):
```

Comment:
perfect. I need to update the login function and I will do that hopefully tonight so that tests work.